langchain/embeddings/huggingface.py

```diff
@@ -48,30 +48,38 @@ class Config:
 
         extra = Extra.forbid
 
-    def embed_documents(self, texts: List[str]) -> List[List[float]]:
+    def embed_documents(
+        self, texts: List[str], normalize_embeddings: bool = True
```

Comment:
we shouldn't change this function signature since it's defined in parent class (`Embeddings`). probably we should put `normalize_embeddings` on the object itself as an instance attribute

so something like
```python
class HuggingFaceEmbeddings:
  normalize_embeddings: bool = True
  ...

  def embed_documents(self, texts):
    embeddings = self.client.encode(text, normalize_embeddings=self.normalize_embeddings)
    ...
```